src/box/lua/config/instance_config.lua

```diff
@@ -1694,7 +1694,98 @@ return schema.new('instance_config', schema.record({
                 end
             end,
         }),
-    })
+    }),
+    audit_log = enterprise_edition(schema.record({
+        to = enterprise_edition(schema.enum({
+            'devnull',
+            'file',
+            'pipe',
+            'syslog',
+        }, {
+            default = 'devnull',
+        })),
+        file = enterprise_edition(schema.scalar({
+            type = 'string',
+            default = 'var/log/{{ instance_name }}/audit.log',
+        })),
```

Comment:
It should be supported by the mkdir applier, a comment should be added here. See `log.file` for example.

Reply:
We should actually create a directory in applier/mkdir.lua. I guess it'll be just copy-paste of a code block that serves `log.file`.